controllers/front/StoresController.php

```diff
@@ -286,7 +286,7 @@ public function setMedia()
 
         if (!Configuration::get('PS_STORES_SIMPLIFIED')) {
             $default_country = new Country((int)Tools::getCountry());
-            $this->addJS('http'.((Configuration::get('PS_SSL_ENABLED') && Configuration::get('PS_SSL_ENABLED_EVERYWHERE')) ? 's' : '').'://maps.google.com/maps/api/js?sensor=true&region='.substr($default_country->iso_code, 0, 2));
```

Comment:
I think you were heavy-handed, you removed `&region` also.
Could you edit your PR?
Also, your commit doesn't respect our[ commit standards](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message), could you edit your commit message?